packages/stepper/src/components/Step.js

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import Container from "../styledComponents/Step";
+import statusTypes from "../constants/statusTypes";
+
+export default class Step extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick(event) {
```

Comment:
wouldnt it be better to do this wrapping on the parent component? and then just pass the onClick property to the current element?

Reply:
one advantage of this wrapping belonging to the parent is that we can remove the need for an index property and make this step component simpler